pkg/common/config.go

```diff
@@ -39,8 +39,8 @@ var (
 
     CsiVersion = "1"
 
-    // The list of export path prefixes to try to use, in order, when mounting to a data portal with NFS v3
-    DefaultDataPortalMountPrefixes = [...]string{"/mnt/data-portal", "/"}
+    // The list of export path prefixes to try to use, in order, when mounting to a data portal
+    DefaultDataPortalMountPrefixes = [...]string{"/mnt/data-portal", "/", ""}
```

Comment:
@ameade, can you mention why this was needed?

Reply:
It's not actually. I was seeing double //s for awhile